eva_sub_cli/semantic_metadata.py

```diff
@@ -184,15 +190,45 @@ def _validate_biosample_against_checklist(self, sample_data, json_path, accessio
             # Successful response returns the sample object, error response has a list of error objects.
             if isinstance(response, list):
                 for error_dict in response:
+                    error_property = ''
+                    # Retrieve the property the error applied to in the datapath
+                    if 'dataPath' in error_dict:
+                        if error_dict['dataPath'].startswith('/characteristics/'):
+                            error_property = error_dict['dataPath'].split('/')[2]
+                        else:
+                            error_property = error_dict['dataPath']
+                    # Define the error
+                    error_list = []
                     if accession:
-                        self.add_error(json_path, f'Existing sample {accession} {error_dict["errors"][0]}')
-                    else:
-                        self.add_error(json_path, error_dict["errors"][0])
+                        error_list.append(f'In existing sample {accession}')
```

Comment:
Is there a prefix we can use that will work for all error types? Neither `In existing sample SAME00004` or `Existing sample SAME00004` really work for both `<prefix> must have required property` and `<prefix> collection date must match pattern`.

Maybe something more flexible like `Error validating existing sample SAME00004:`? I don't want to have to micromanage ever-changing error messages from other services, but also users will still complain to us if the messages are confusing (even if they're not our fault)...

Reply:
Done.